selendroid-standalone/src/main/java/io/selendroid/standalone/android/AndroidSdk.java

```diff
@@ -230,4 +236,40 @@ public static void setAndroidSdkVersion(String androidSdkVersion) {
   public static void setBuildToolsVersion(String buildToolsVersion) {
     sBuildToolsVersion = buildToolsVersion;
   }
+  public static File avdManager() {
+    if (sAvdManager != null) {
+      return new File(sAvdManager + platformExecutableSuffixExe());
+    }
+    else {
+      return android();
+    }
+  }
+
+  public static int getAndroidVersionNumber() {
+    if (sAndroidSdkVersion != null) {
+      try {
+        return Integer.parseInt(sAndroidSdkVersion);
```

Comment:
The android version override is meant to be a folder name in `$ANDROID_HOME/platforms/` so it should be of the form `android-xx` so you should still extract the number using `PLATFORM_VERSION_REGEX`